### DISCUSSION TITLE
Mode 10: Blade fill level

### DISCUSSION THREAD

**Discussion by SvenVoeten** [Courseplay/courseplay#7646] [Q&A] [answered]
Hi,

I'm playing with mode 10 to fill up my bunker, it's working pretty good and the bunker is filling up really nice.
But it is going really slow due to the fact that the blade is not filling up entirely when pushing the chaff backwards .

The blade barely go's above 30% fill level as you can see on the picture below.
If it would fill up completely then the process of filling up the bunker would go much faster.
Am I doing something wrong?

![image](https://user-images.githubusercontent.com/65830467/144758360-731861ad-a318-44ba-94df-5c21f5e6d057.png)

![image](https://user-images.githubusercontent.com/65830467/144758463-4ed7ceaf-e8ea-4ab1-aa57-5b7317e7b17a.png)

**Comment by Tensuko** [comment, marked as answer]
You don't do anything wrong, that is just the way it works.
Try to move your Vehicle yourself with a full push and 100% shield.
It is less effective. (if you can move on anyway)
